chapel.html.markdown

```diff
@@ -310,6 +311,18 @@ stringSet += "a"; // Redundant add "a"
 stringSet -= "c"; // Remove "c"
 writeln( stringSet ); 
 
+// Both ranges and domains can be sliced to produce a range or domain with the
+// intersection of indices
+var rangeA = 1.. ; // range from 1 to infinity
+var rangeB =  ..5; // range from negative infinity to 5
```

Comment:
So per the spec (section 18.2) the 'boundness' of the range is stored as an enum of bounded, boundedLow, boundedHigh, boundedNone.
Im not sure what the implementation is (though Im going to ask around now) but If one were to 
```c
for i in max(int(8))-1.. do something(i);
```
it will just overflow to min(int(8)) and keep going in an infinite loop.
I guess infinity means a lot of things.